app/Http/Controllers/Playlist/PlaylistsController.php

```diff
@@ -111,8 +111,8 @@ public function index(Request $request)
         }
 
         if ($featured) {
-            $cache_string = generateCacheString('v4_playlist_index', [$show_details, $featured, $sort_by, $sort_dir, $limit, $show_text]);
-            $playlists = \Cache::remember($cache_string, now()->addDay(), function () use ($show_details, $user, $featured, $sort_by, $sort_dir, $limit, $show_text) {
+            $cache_params = [$show_details, $featured, $sort_by, $sort_dir, $limit, $show_text];
+            $playlists = cacheRemember('v4_playlist_index', $cache_params, now()->addDay(), function () use ($show_details, $user, $featured, $sort_by, $sort_dir, $limit, $show_text) {
```

Comment:
hi @mgonzalezs.  this (and many other snippets) look like reorganization, not change in functionality.  where/how does this PR ensure that the compute-intensive queries for featured playlists (observed during 5/5 downtime) will not unnecessarily hammer the database (eg via memcached)?

Reply:
That is correct this PR reorganizes the Cache functions.

The intention of this PR is to verify that the cache is working, and is working if the server is configured well, so we have now `http://dbp.test/status/cache` endpoint to verify that.

In conclusion, If the playlists are not being cached means that the other endpoints also are not working.

The improvement of playlists will be done on https://fullstacklabs.atlassian.net/browse/FCBH-1830